hc/front/tests/test_add_pushbover.py

```diff
@@ -1,7 +1,8 @@
 from django.test.utils import override_settings
```

Comment:
Thank you on the feedback, was a bit hesitant on changing the wrongly named file at first but now have clarity I shall rename the file to `test_add_pushover.py`